app/translations/en.json

```
@@ -640,6 +641,15 @@
                     "clone": "Clone filter"
                 }
             },
+            "modal": {
+                "id": "Filter id",
+                "idMissing": "Please provide filter id",
```

Optional:

I would use proper English here since it looks like a sentence

```suggestion
                "idMissing": "Please provide the filter id",
```

or

```suggestion
                "idMissing": "Please provide a filter id",
```